pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -49,6 +46,7 @@ import (
 	st "k8s.io/kubernetes/pkg/scheduler/testing"
 	tf "k8s.io/kubernetes/pkg/scheduler/testing/framework"
 	"k8s.io/kubernetes/pkg/scheduler/util/assumecache"
+	"k8s.io/kubernetes/test/utils/ktesting"
```

Comment:
This package includes setting up the '-v' command line flag, replacing `"k8s.io/klog/v2/ktesting/init"`.